package.json

```diff
@@ -118,7 +118,9 @@
     ]
   },
   "files": [
+    "config.d.ts",
```

Comment:
This file [already exists](https://github.com/cortexapps/backstage-plugin/blob/master/config.d.ts), but we were neither exposing it from the package nor marking it as the definition for the config schema for backstage to use.